src/blocks/scratch3_sensing.js

```diff
@@ -74,10 +74,18 @@ class Scratch3SensingBlocks {
 
     getMonitored () {
         return {
-            sensing_answer: {},
-            sensing_loudness: {},
-            sensing_timer: {},
-            sensing_current: {}
+            sensing_answer: {
+                getId: () => 'answer'
+            },
+            sensing_loudness: {
+                getId: () => 'loudness'
+            },
+            sensing_timer: {
+                getId: () => 'timer'
+            },
+            sensing_current: {
+                getId: (_, param) => `current_${param}`
```

Comment:
We should probably add a note here about why this id is different from what's in the default toolbox xml. E.g. we want monitors to be imported as separately from sb2, but we want the block in the toolbox to only ever create one monitor.

I also just realized that this getId function is specific to sb2 now, (e.g. I won't be able to use it in sb3 deserialization) but we can potentially cross that bridge when we get there.